### ISSUE TITLE
Input Radio Group touched and focus Accessibility Issue

### ISSUE THREAD

**Issue by bergerb** [angular/components#18543]
#### What are you trying to do?

Understand the relationship of touched and focus on a input radio group that is linked to a Reactive Form.  This is in relation to WCAG Accessibility and Usability.

The focus is set on the input but the touched isn't fired until after the focus is lost.  This behavior can cause an unexpected behavior in a keyboard only environment.

In this scenario the keyboard user needs to use the arrow keys to navigate the options provided.  The user cannot go to the next option without a delay of checking the value of touched.

I'm not sure this is the correct place for this question.

#### What troubleshooting steps have you tried?

Adding a delay does fix this problem, but it also causes some unintended consequences such as the jumping of focus.

What are you seeing that does not match your expectations?   

There is a delay in the firing of touched when using keyboard navigation.  Is this expected behavior?

#### Reproduction

https://stackblitz.com/edit/angular-b36sfr

Keyboard only mode

Steps to reproduce:
1.  Tab through selections on first field group (incorrect behavior)
2.  Tab through second radio button group (correct behavior)
  
#### Environment

- Angular: 8.0
- CDK/Material: 8.0
- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu):  Windows

**Comment by jelbourn** [comment]
It looks like your StackBlitz isn't actually using any Angular Material components (it's the native radio button). At a high level, it seems like you're looking for advise on implementing an auto-focus directive; better avenues for that might be the Angular community discord or StackOverlow.
